frontend/www/js/omegaup/components/login/Signin.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <div>
+    <omegaup-login
+      :facebook-u-r-l="facebookURL"
```

Comment:
```suggestion
      :facebook-url="facebookUrl"
```

del otro lado puedes usar `facebookUrl` en vez de `facebookURL`.

Reply:
Listo